userspace/libsinsp/parsers.h

```diff
@@ -108,6 +108,7 @@ class sinsp_parser
 	void parse_close_exit(sinsp_evt* evt);
 	void parse_thread_exit(sinsp_evt* evt);
 	inline bool detect_and_process_tracer_write(sinsp_evt *evt, int64_t retval, ppm_event_flags eflags);
+	void parse_file_related_exit(sinsp_evt* evt);
```

Comment:
```suggestion
	void parse_fspath_related_exit(sinsp_evt* evt);
```
?

Reply:
Sure--I wasn't actually thinking of it being specifically related to the fspath filtercheck, but I guess it really is only used for this, so okay.